src/components/Header.jsx

```diff
@@ -1,17 +1,32 @@
-import React from 'react';
-import { Nav, Navbar } from 'react-bootstrap';
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { useState } from 'react';
+import { Button, Nav, Navbar } from 'react-bootstrap';
+import * as FaIcons from 'react-icons/fa';
+import SearchJob from './SearchJob';
 
 function Header() {
+	const [sidebar, setSidebar] = useState(false);
+
+	const showSidebar = () => setSidebar(!sidebar);
```

Comment:
I'd just write this inline with the `onClick` but you should be passing in `true` instead of `!sidebar`.